### ISSUE TITLE
history test times out every time

### ISSUE THREAD

**Issue by mqudsi** [fish-shell/fish-shell#7193]
Running under a pretty clean install of Elementary OS (based off of Ubuntu 18.04), the pexpect-based history test fails each and every time.

```
Testing file pexpects/history.py ... Failed to match pattern: prompt 1
history.py:30: timeout from expect_prompt()


Escaped buffer:
\x1b[30m\x1b(B\x1b[m\x1b[2m⏎\x1b(B\x1b[m                                                                               \r⏎ \r\x1b[K\x1b]7;file://studiog3/home/mqudsi/fish-shell/build/tests\x07\x1b[30m\x1b(B\x1b[mprompt 1>\x1b[K

When written to the tty, this looks like:
<-------
prompt 1>
------->

Last 10 messages:

```

**Comment by faho** [comment]
What terminal is this in?

**Comment by mqudsi** [comment]
Pantheon terminal.

**Comment by ridiculousfish** [comment]
Huh, looks like the pattern is in the buffer, not sure why it would fail to match

**Comment by faho** [comment]
I'm guessing the regex mismatches some of the sequence?

For me Pantheon Terminal fails on the CWD OSC, not this. So let's disable that in the tests and see what happens.

I see @mqudsi has already disabled the snippet reading (which triggered a helpful job completion notification which just happened to show up once per line pexpect sends which led to 50 notifications in one second which managed to crash plasmashell for me).

**Comment by faho** [comment]
@mqudsi That fixes it for me with Pantheon Terminal 5.5.2 on archlinux, please holler if it doesn't work on the real thing.

**Comment by mqudsi** [comment]
Confirmed working on Pantheon under Elementary OS as well. Thank you all.
